src/components/SymbolTabs/SymbolTabs.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="symbol-tab-container" :class="[direction === 'horizontal' ? 'horizontal' : 'vertical']">
+    <div class="symbol-tab-nav">
+      <div
+        v-for="({ label }, index) in tabPaneNavigators"
+        :key="index"
+        class="nav-item"
+        :class="[index === activeNavIndex ? 'active-item' : 'inactive-item']"
+        @click="onTabChange(index)"
+      >
+        <span>{{ label }}</span>
```

Comment:
This label should be internationalized

Reply:
NO，this is a UI component.
Keep it single

Reply:
I meant the label is never translated

Reply:
Then it should be translated and then transferred to label. like this 
```
<SymbolTabPane :label="$t('network_settings')">
       <FormNodeEdit @onCloseModal="isShowModal = false" />
</SymbolTabPane>
```